src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxTreeExtensions.cs

```diff
@@ -166,6 +164,43 @@ public static bool IsPrimaryFunctionExpressionContext(this SyntaxTree syntaxTree
                 syntaxTree.IsSizeOfExpressionContext(position, tokenOnLeftOfPosition);
         }
 
+        public static bool IsTypeOfExpressionContext(this SyntaxTree syntaxTree, int position, SyntaxToken tokenOnLeftOfPosition)
```

Comment:
Same for this file, just moving existing code.

Reply:
So the context query stuff was intentionally split out :-/. I'm not thrilled it was merged in

Reply:
So the context query stuff was intentionally split out :-/. I'm not thrilled it was merged in